src/views/teachers/landing/landing.jsx

```diff
@@ -82,19 +82,25 @@ var Landing = injectIntl(React.createClass({
                         <h3 id="guides-header"><FormattedMessage id="teacherlanding.guidesTitle" /></h3>
                         <FlexRow className="guides-and-tutorials">
                             <div>
-                                <img src="/svgs/teachers/resources.svg" alt="resources icon" />
+                                <a href="https://scratch.mit.edu/help/">
```

Comment:
I think this can be a relative url, like it is in https://github.com/LLK/scratch-www/blob/develop/src/views/teachers/landing/l10n.json#L17